src/test/javascript/spec/component/lecture-unit/exercise-unit/exercise-unit.component.spec.ts

```diff
@@ -67,7 +58,7 @@ describe('ExerciseUnitComponent', () => {
             By.directive(CourseExerciseRowStubComponent),
         ).componentInstance;
 
-        expect(courseExerciseRowStubComponent.course).to.equal(exerciseUnitComponent.course);
-        expect(courseExerciseRowStubComponent.exercise).to.equal(exerciseUnitComponent.exerciseUnit.exercise);
+        expect(courseExerciseRowStubComponent.course).toEqual(exerciseUnitComponent.course);
```

Comment:
You replaced to.equal with to.Be in the other file, and due to our documentation I think it wouldn't be bad to stick to one, i.e. to.Be. What do you think?

Reply:
``toBe`` and ``toEqual`` aren't the same thing; ``toBe`` works good for enum values, primitives and strings etc. as it is using ``Object.is(a, b)`` under the hood, but ``toEqual`` performs a deep quality check on objects, which is why it's used here.